packages/amplify_auth_cognito/ios/Classes/Utils/UserAttributeDeserialization.swift

```diff
@@ -50,6 +50,14 @@ func createAuthUserAttributeKey(keyName: String) -> AuthUserAttributeKey {
     case "preferred_username":
         return AuthUserAttributeKey.preferredUsername
     default:
-        return AuthUserAttributeKey.custom(keyName)
+        // amplify-ios does not currently include enums for these keyNames
+        let unknownKeys = ["profile", "updated_at", "website", "zoneinfo"]
+        if (unknownKeys.contains(keyName)) {
+            return AuthUserAttributeKey.unknown(keyName)
+        } else {
+            // amplify-ios prepends 'custom:' to custom keys
+            let customKeyName = keyName.starts(with: "custom:") ? String(keyName.dropFirst(7)) : keyName
```

Comment:
Adding comment again for visibility.
___

This change will make amplify-flutter consistent on iOS and Android. On both platforms, prepending custom: will be optional with this change. Amplify-android requires that the value be prepended with custom: so we add it if it is missing. This is essentially the opposite. amplify-ios requires that it is not present, so we remove it.

Note: When the key is created in Cognito, Cognito does not allow for users to prepend custom: to the key name since it will be added automatically. There should be no risk of an edge case scenario where a users key prepended custom: during creating, resulting in a key like custom:custom:my_key.

Reply:
I think thats fair. Something like this?
```suggestion
            let customKeyPrefix = "custom:"
            let customKeyName = keyName.starts(with: customKeyPrefix) ? String(keyName.dropFirst(customKeyPrefix.count)) : keyName
```